cmd/nanobus/main.go

```diff
@@ -91,9 +91,8 @@ func (c *defaultRunCmd) Run() error {
 }
 
 type runCmd struct {
-	DeveloperMode bool `name:"developer-mode" help:"Enables developer mode."`
-	// BusFile of the application as a configuration file or OCI image reference.
-	BusFile string `name:"bus" default:"bus.yaml" help:"The application configuration or OCI image reference"`
+	Target        string `arg:"positional" required:"" help:"The application configuration yaml file or OCI image reference."`
```

Comment:
Nit: `arg` does not require a value. Its existence is like a flag for the CLI library.

We also want `target` to be optional with a default value of `bus.yaml` so that `nanobus run` and `nanobus run <target>` both work. So I think the struct tags should be:

```
arg:"" default:"bus.yaml" help:"The application configuration yaml file or OCI image reference."
```

Reply:
I had to have `arg:"positional" to get this to work.  Without it, it complained that there were no positional arguments.